gitlab/v4/objects/__init__.py

```diff
@@ -1210,6 +1210,7 @@ class GroupMergeRequestManager(ListMixin, RESTManager):
         "source_branch",
         "target_branch",
         "search",
+        "wip",
```

Comment:
Thanks for the addition @jlpospisil! One small note. You added this to the `MergeRequestManager` (global) and `GroupMergeRequestManager` (groups). I think you can also add it to `ProjectMergeRequestManager`  (projects) per the API docs.

It seems like `wip` is not documented as a param in group merge requests in https://docs.gitlab.com/ee/api/merge_requests.html#list-group-merge-requests, but it probably works. Did you already use it this way?

Reply:
I use it via the UI all the time which is what led me to try it via this library.  The possible values are `yes` or `no`.  I primarily use `wip=no` to get all non WIP MRs.  This will filter out anything that is labelled as `WIP` or where the description starts with `WIP`.  I verified that it worked as expected for my use case.  I specifically used the `GroupMergeRequestManger`.  I did add the filter to the group manager as well.

Reply:
Great, thanks! I wasn't sure if it was covered by the Groups API since the GitLab docs didn't have it there. Seems to happen sometimes with their documentation :)